### PR TITLE
Maintenance PR - React 16. peerDependencies added.

### PR DESCRIPTION
**Pull request by export-mike**
Hi thanks for the package 👍 

I've updated to use peer dependencies, updated to allow use of react 16 and added react to storybook-router dev deps to allow yarn/npm link within examples to test locally

Any questions let me know, happy to help. 

Cheers

### CODE REVIEW

**Comment by ndelangen** [comment]
LGTM!

File: package.json [hunk 1]
```diff
@@ -21,20 +21,24 @@
...
   "devDependencies": {
     "babel-cli": "^6.24.1",
     "babel-preset-es2015": "^6.24.1",
     "eslint": "^3.19.0",
     "eslint-plugin-react": "^7.0.1",
+    "react": "^16.0.0",
     "react-scripts": "1.0.7"
   },
+  "peerDependencies": {
+    "react": "^0.14.7 || ^15.0.0 || ^16.0.0"
```
Comment (gvaldambrini) [inline package.json]: With the exception of adding the new major version of React for me moving the React dependency from `dependencies` to `peerDependencies` does not make any difference. Even if I'd like to test modifications to this package using `npm link` I cannot see a direct relationship of those things and I still have the error:
```
Uncaught Error: Accessing nonexistent addons channel
```
Maybe @ndelangen has an explaination for that.

Comment (export-mike) [reply package.json]: adding it to peer dependencies ensures consumers do not run the app with multiple version of react. Its a recommended way of bulding third-party components. I've moved the local version of react for testing locally to devDependencies.

File: package.json [hunk 2]
```diff
@@ -21,20 +21,24 @@
     "vue-router"
   ],
   "dependencies": {
+    "@storybook/addon-actions": "^3.2.10",
+    "global": "^4.3.2",
```
[the remark this answers is not in the document]
Comment (export-mike) [reply package.json]: global was being required and it was not in the dependencies list. I was getting build errors.

**Comment by export-mike** [comment]
see: https://github.com/gvaldambrini/storybook-router/pull/18

**Pull request closed without merging**